images-list

```diff
@@ -130,6 +134,10 @@ ghcr.io/banzaicloud/logging-operator rancher/mirrored-banzaicloud-logging-operat
 ghcr.io/banzaicloud/logging-operator rancher/mirrored-banzaicloud-logging-operator 3.17.1
 ghcr.io/banzaicloud/logging-operator rancher/mirrored-banzaicloud-logging-operator 3.17.3
 ghcr.io/banzaicloud/logging-operator rancher/mirrored-banzaicloud-logging-operator 3.17.4
+ghcr.io/epinio/epinio-server rancher/epinio-epinio-server 0.7.1
+ghcr.io/epinio/epinio-server rancher/mirrored-epinio-epinio-server 0.7.1
+ghcr.io/epinio/epinio-ui rancher/epinio-epinio-ui v0.6.2-0.0.1
+ghcr.io/epinio/epinio-ui rancher/mirrored-epinio-epinio-ui v0.6.2-0.0.1
```

Comment:
Same with these. We don't really need to add the double `epinio-epinio` to these.

Reply:
Oh I see. That's a good point.